src/erlfmt_scan.erl

```diff
@@ -235,7 +235,14 @@ drop_initial_white_space(Rest) ->
 
 -spec stringify_tokens([erl_scan:token()]) -> string().
 stringify_tokens(Tokens) ->
-    lists:flatmap(fun erl_scan:text/1, Tokens).
+    lists:flatmap(fun stringify_token/1, Tokens).
+
+-spec stringify_token(erl_scan:token()) -> string().
+stringify_token(Token) ->
+    case erl_scan:text(Token) of
+        undefined -> "";
```

Comment:
In what case does this happen? This is used when skipping formatting and I'm afraid it might end up with some inconsistencies in this case.
I'd like to understand when `undefined` is a case here so we can properly test it

Reply:
Hm, I though this was triggered by the sigil suffix (when empty). But it seems by implementing the sigils correctly, they no longer cause it to produce `undefined`. I'll remove the change.